examples/BUILD.gn

```diff
@@ -7,7 +7,7 @@ group("examples") {
 
   deps = [
     "//examples/demo_launcher",
-    "//examples/fitness",
+    #"//examples/fitness",
```

Comment:
I need to update playfair to understand the new package names.  This circular dependency is no good.